test/Config.hs

```diff
@@ -0,0 +1,14 @@
+module Config (testConfig) where
+
+import Test.Hspec
+import Crocodealer.Config
+import Crocodealer.Core.Label
+
+testConfig :: Config -> SpecWith ()
+testConfig config = 
+    describe "Config.loadConfig" $ 
+        it "Should load the sample configuration correctly" $ do
+            configUsername config `shouldBe` Just "my-user"
+            configRepository config `shouldBe` Just "my-org/my-repo"
+            configIgnoredRepositories config `shouldBe` [Repo {unRepo = "my-ignored-repo"},Repo {unRepo = "my-other-ignored-repo"}]
+            configLabelRules config `shouldBe` [Create (Label {labelName = Name {unName = "label1"}, labelDescription = "a cool label", labelColor = HexColor {unHexColor = "#000000"}}),Override (Name {unName = "old label name"}) (Name {unName = "new label name"})]
```

Comment:
Instead of comparing each field, let's create a single value `exampleConfig :: Config` and the test will just be:

```haskell
 it "Should load the sample configuration correctly" $
    config `shouldBe` exampleConfig
```